src/Maestro/Maestro.Data/Migrations/20190415232152_indexAssetNameVersion.cs

```diff
@@ -0,0 +1,54 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace Maestro.Data.Migrations
+{
+    public partial class indexAssetNameVersion : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.AlterColumn<string>(
+                name: "Version",
+                table: "Assets",
+                maxLength: 400,
```

Comment:
It should be smaller that this. We might want to create a multi-column index, which this will prevent.  It's generally better to make columns smaller.  Smaller database, smaller indices, more flexibility.  You said the current max was 72, right? Maybe make this 100 and call it a day.  It's generally easier to make these larger later than the inverse.

Reply:
Version DEFINITIELY doesn't need to be 400 characters. Just far too large a size for the version.

Reply:
The current max name is 121, and version is 67. I could go to 150?

Reply:
150 and 75/100 seems fine.  I would expect the versions to be smaller (that one is almost certainly an error, where someone concatenated their name+version).  I believe the size of the index on disk will be whatever the "max" is, so the smaller the better.